v3/src/components/graph/utilities/graph-utils.ts

```diff
@@ -382,7 +382,7 @@ export function setPointSelection(props: ISetPointSelection) {
     .style('stroke-width', defaultStrokeWidth)
     .style('stroke-opacity', defaultStrokeOpacity)
 
-  const selectedDots = dots.selectAll('.graph-dot-highlighted')
+  const selectedDots = select(dotsRef.current).selectAll('.graph-dot-highlighted')
```

Comment:
```suggestion
  const selectedDots = selectDots(dotsRef.current, true)
```
In this case I think it would make sense to use the new API, which was probably what I originally intended to do and then just forgot to do so.